bin/redshift/local_delete_aws_redshift.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+cd "${DIR}/../.." || exit 1
```

Comment:
> `|| exit 1`

this reminds me that all scripts should start with

```bash
set -xeuo pipefail
```

then you don't need exit code checks on individual statements like here